server/src/test/java/org/elasticsearch/repositories/blobstore/BlobStoreRepositoryTests.java

```diff
@@ -248,40 +247,28 @@ public void testBadChunksize() throws Exception {
                 .get());
     }
 
-    public void testFsRepositoryCompressDeprecated() {
-        final Path location = ESIntegTestCase.randomRepoPath(node().settings());
-        final Settings settings = Settings.builder().put(node().settings()).put("location", location).build();
-        final RepositoryMetaData metaData = new RepositoryMetaData("test-repo", REPO_TYPE, settings);
-
-        Settings useCompressSettings = Settings.builder()
-            .put(node().getEnvironment().settings())
-            .put(FsRepository.REPOSITORIES_COMPRESS_SETTING.getKey(), true)
-            .build();
-        Environment useCompressEnvironment =
-            new Environment(useCompressSettings, node().getEnvironment().configFile());
-
-        new FsRepository(metaData, useCompressEnvironment, null);
-
-        assertWarnings("[repositories.fs.compress] setting was deprecated in Elasticsearch and will be removed in a future release!" +
-            " See the breaking changes documentation for the next major version.");
```

Comment:
@henningandersen maybe we should add a small note, given that we kept logging this?

Reply:
Yes, makes sense, thanks.